kafka-admin/src/main/java/io/strimzi/admin/kafka/admin/KafkaAdminService.java

```diff
@@ -38,22 +50,28 @@
 
                 final TypeDefinitionRegistry typeDefinitionRegistry = schemaParser.parse(userInputStream);
                 p.complete(typeDefinitionRegistry);
-            }
-            catch (Exception exc) {
+            } catch (Exception exc) {
                 p.fail(exc);
             }
         }, ar -> {
-            final TypeDefinitionRegistry schema = (TypeDefinitionRegistry) ar.result();
+                final TypeDefinitionRegistry schema = (TypeDefinitionRegistry) ar.result();
 
-            final RuntimeWiring query = RuntimeWiring.newRuntimeWiring()
-                .type("Query", typeWiring -> typeWiring
-                    .dataFetcher("topic", new VertxDataFetcher<>(TopicHandler::getTopic))
-                    .dataFetcher("topicList", new VertxDataFetcher<>(TopicListHandler::getTopicList))
-                )
-                .build();
+                AdminClientProvider acp = new AdminClientProvider(vertx, config);
+                acp.open();
+                // todo close acp
```

Comment:
what about this todo? are working on it? why should we close the acp here?

Reply:
I am not sure whether we need to close AdminClient at some point.